openvpn_status/utils.py

```diff
@@ -21,7 +22,10 @@ def parse_time(time):
     """Parses date and time from input string in OpenVPN logging format."""
     if isinstance(time, datetime.datetime):
         return time
-    return datetime.datetime.strptime(time, DATETIME_FORMAT_OPENVPN)
+    try:
+        return datetime.datetime.strptime(time, DATETIME_FORMAT_OPENVPN_F1)
+    except Exception as e:
```

Comment:
It may be better to catch `ValueError` only (`except Value:`). [Ref:](https://docs.python.org/3.11/library/datetime.html#datetime.datetime.strptime)

> ValueError is raised if the date_string and format can’t be parsed by time.strptime() or if it returns a value which isn’t a time tuple.